cmd/machine-healthcheck/main.go

```diff
@@ -19,11 +19,11 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/manager/signals"
 )
 
-// The default durations for the leader electrion operations.
+// The default durations for the leader election operations.
 var (
-	leaseDuration = 120 * time.Second
-	renewDealine  = 110 * time.Second
-	retryPeriod   = 90 * time.Second
+	leaseDuration = 137 * time.Second
+	renewDealine  = 107 * time.Second
+	retryPeriod   = 26 * time.Second
```

Comment:
Would it be better to import the values from the helpers package here? 
```suggestion
	leaseDuration = helpers.LeaseDuration
	renewDealine  = helpers.RenewDeadline
	retryPeriod   = helpers.RetryPeriod
```

Reply:
In this case we have to import "github.com/openshift/machine-api-operator/cmd/machinea-api-operator", I don't think it's a good practice.
It would be better to create a file in pkg/utils and use it everywhere.